index.js

```diff
@@ -135,7 +127,7 @@ const legacyNaming = {
 const protections = {}
 for (const key in legacyNaming) {
     let newConfig
-    if (!defaultConfig.features[key] || defaultConfig.features[key].state !== 'enabled') {
```

Comment:
so previously for things like 'beta' (that we don't use just yet AFAIK) this would be true and now it's false, is that expected?

Reply:
I don't think we should be coding for this beta stuff as there's no supporting code. The format if forwards compatible and that's all we needed.


I've removed all this code as now we've simplified things we can just simplify the transform here.

Reply:
Yup, fair point about the beta 👍